files/default/README.md

```diff
@@ -0,0 +1,14 @@
+# Unit Testing
+
+Make a python virtual environment, install the requirements with:
+
+    mkvirtualenv hatool -p python2
```

Comment:
I'm removing those steps, and just assuming the reader is capable of creating an environment.